### DISCUSSION TITLE
有没有图床功能？

### DISCUSSION THREAD

**Discussion by ChenJinShui** [spencerwooo/onedrive-vercel-index#290] [Q&A] [answered]
有没有图床功能？

**Comment by spencerwooo** [comment, marked as answer]
https://docs.spencerwoo.com/hosting-images-files-directly

**Reply by leftchest** [reply]
https://drive.spencerwoo.com/api?path=/%F0%9F%A5%9F%20Some%20test%20files/Previews/image/minion.png&raw=true
地址后面都带有raw=true如何添加到notion中使用呢？恳请指导

**Reply by spencerwooo** [reply]
<img width="829" alt="Screenshot 2022-01-23 at 15 45 20" src="https://user-images.githubusercontent.com/32114380/150669318-8f7ccf34-6489-4a4f-961d-e0a4e808909e.png">

这样就可以吧？

**Reply by leftchest** [reply]
图片可以，但是mp4为什么不行呢，比如您网盘mp4：https://drive.spencerwoo.com/api?path=/%F0%9F%A5%9F%20Some%20test%20files/Videos%20and%20music/Story%20Teaser%20-%20Through%20the%20Eyes%20of%20a%20Dragon%20%20-%20Genshin%20Impact.mp4&raw=true
如何插入notion呢？

**Reply by leftchest** [reply]
图片网上很多图床可以用，主要想把网盘视频插入notion用，像插入图片那样不成功，恳请指导

**Reply by spencerwooo** [reply]
可能是 Notion 支持的视频仅有各大视频平台（比如 YouTube 和 Vimeo）以及自己上传的视频，并不支持这样 embed 一个视频文件直链

**Reply by leftchest** [reply]
插入视频那里和图片一样，也是有个embed link选项，我把OneDrive共享链接在其他平台转成以.mp4结尾的直链后是可以正常插入使用的，但是这个项目的视频地址以raw=true结尾就不可以了，又没什么变通的办法呢，比如地址能以.mp4结尾

**Reply by myl7** [reply]
#306 这样你试试？

**Reply by zkeq** [reply]
> 插入视频那里和图片一样，也是有个embed link选项，我把OneDrive共享链接在其他平台转成以.mp4结尾的直链后是可以正常插入使用的，但是这个项目的视频地址以raw=true结尾就不可以了，又没什么变通的办法呢，比如地址能以.mp4结尾

仅仅地址以`.mp4`结尾的话,直接把`raw=true`移到前面不就行了吗...
但是Notion还是无法识别,大概是地址在此会跳转的缘故...

`https://drive.spencerwoo.com/api?raw=true&path=/`

https://drive.spencerwoo.com/api?raw=true&path=/%F0%9F%A5%9F%20Some%20test%20files/Videos%20and%20music/Story%20Teaser%20-%20Through%20the%20Eyes%20of%20a%20Dragon%20%20-%20Genshin%20Impact.mp4

**Reply by leftchest** [reply]
> #306 这样你试试？

感谢指导，是把直链地址里面的api？改为api/name/<filename>?吗？试了下不行应该不是这个意思，恳请详细指导下小白级别的网友

**Reply by leftchest** [reply]
> > 插入视频那里和图片一样，也是有个embed link选项，我把OneDrive共享链接在其他平台转成以.mp4结尾的直链后是可以正常插入使用的，但是这个项目的视频地址以raw=true结尾就不可以了，又没什么变通的办法呢，比如地址能以.mp4结尾
> 
> 仅仅地址以`.mp4`结尾的话,直接把`raw=true`移到前面不就行了吗... 但是Notion还是无法识别,大概是地址在此会跳转的缘故...
> 
> `https://drive.spencerwoo.com/api?raw=true&path=/`
> 
> https://drive.spencerwoo.com/api?raw=true&path=/%F0%9F%A5%9F%20Some%20test%20files/Videos%20and%20music/Story%20Teaser%20-%20Through%20the%20Eyes%20of%20a%20Dragon%20%20-%20Genshin%20Impact.mp4

谢谢，确实是，楼上好像给出了办法，奈何自己太小白
